pkg/plugin/v3/scaffolds/internal/templates/config/api/hub_conversion.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &HubConversion{}
+
+type HubConversion struct { // nolint:maligned
+	file.TemplateMixin
+	file.MultiGroupMixin
+	file.BoilerplateMixin
+	file.ResourceMixin
+
+	HubVersion string
+}
+
+func (c *HubConversion) SetTemplateDefaults() error {
+	c.Resource.Version = c.HubVersion
+	if c.Path == "" {
+		if c.MultiGroup {
+			c.Path = filepath.Join("apis", "%[group]", "%[version]", "%[kind]_conversion.go")
+		} else {
+			c.Path = filepath.Join("api", "%[version]", "%[kind]_conversion.go")
+		}
+	}
+	c.Path = c.Resource.Replacer().Replace(c.Path)
+	fmt.Printf("Scaffolding conversion logic for hub version %s in path %s", c.Resource.Version, c.Path)
+	c.TemplateBody = conversionTemplate
+	c.IfExistsAction = file.Error
+
+	return nil
+}
+
+const conversionTemplate = `{{ .Boilerplate }}
+
+package {{ .Resource.Version }}
+
+/*
+Implementing the hub method is pretty easy -- we just have to add an empty
+method called 'Hub()' to serve as a
+[marker](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/conversion?tab=doc#Hub).
+We could also just put this inline in our 'cronjob_types.go' file.
+*/
+
+// Hub marks this type as a conversion hub.
+func (*{{ .Resource.Kind }}) Hub() {}
```

Comment:
This doesn't necessarily need to be a pointer since this method does nothing.
```suggestion
func ({{ .Resource.Kind }}) Hub() {}
```